Sources/Controllers/MessagesViewController.swift

```diff
@@ -341,8 +336,8 @@ fileprivate extension MessagesViewController {
             
             messageCollectionViewBottomInset = afterBottomInset
             
-            if scrollsOnKeyboardFrameChanges {
-                messagesCollectionView.setContentOffset(contentOffset, animated: true)
+            if maintainPositionOnKeyboardFrameChange {
```

Comment:
@SD10 Change from `maintainPositionOnKeyboardFrameChange ` to `maintainPositionOnKeyboardFrameChanged` would be better?

Reply:
Sure, I'll change it when I add the CHANGELOG entry after I merge 👍